libkbfs/interfaces.go

```diff
@@ -1599,6 +1611,10 @@ type BareRootMetadata interface {
 	AreKeyGenerationsEqual(Codec, BareRootMetadata) (bool, error)
 	// GetUnresolvedParticipants returns any unresolved readers and writers present in this revision of metadata.
 	GetUnresolvedParticipants() (readers, writers []keybase1.SocialAssertion)
+	// GetTLFWriterKeyBundleID returns the ID of the writer key bundle.
```

Comment:
Maybe `returns the ID of the externally-stored writer key bundle, or the zero value if this object stores it internally.`